js/about/newprivatetab.js

```diff
@@ -29,6 +38,17 @@ class NewPrivateTab extends React.Component {
           <p className={css(styles.text)} data-l10n-id='privateTabText1' />
           <p className={css(styles.text)} data-l10n-id='privateTabText2' />
           <p className={css(styles.text, styles.italic)} data-l10n-id='privateTabText3' />
+          <h2 className={css(styles.sectionTitle)} data-l10n-id='privateTabSearchSectionTitle' />
```

Comment:
FYI there is a component for section titles (please see: `about:styles#sectionTitle`)